beta-src/src/state/game/game-api-slice.ts

```diff
@@ -143,14 +144,25 @@ export const markMessagesSeen = createAsyncThunk(
     gameID: string;
     seenCountryID: string;
   }) => {
-    const { data } = await getGameApiRequest(
+    const { data } = await postGameApiRequest(
       ApiRoute.MESSAGES_SEEN,
       queryParams,
     );
     return data as string;
```

Comment:
This `as` would be better put inside the `*GameApiRequest` functions, I think. 

Also, since it's just a string that we're looking for, we could check that it's actually a string and throw an error if it isn't, too.

Reply:
I don't know why this `as string` is here - the linter isn't complaining even when I remove it.